blog/2023-10-20-rebranding.mdx

```diff
@@ -0,0 +1,57 @@
+---
+slug: tolgee-rebranding
+title: "Tolgee's Exciting Transformation: A Modern, Fresh Look"
+description: Tolgee rebranding to new colors. We choose pink for it's modern and vibrant tone.
+authors: [sgranat]
+tags: ['brand', 'pink', 'rebranding', 'tolgee']
+---
+
+import { ReviewAuthor } from '../platform/shared/_ReviewAuthor';
+import { ScreenshotWrapper } from '../platform/shared/_ScreenshotWrapper';
+
+We're thrilled to introduce our refreshed brand, including our logo in a vibrant shade of pink, and a modern, user-friendly design for our app and website. We've heard your feedback, and we've taken it to heart.
```

Comment:
I'd rewrite it as following: After a little over 2 years, we decided to freshen up the Tolgee brand, including our logo, website and platform's user interface. We value your suggestions and want to provide the best experience for our users. So, we rolled up our sleeves and gave Tolgee a little refresh to not feel so old-school.